### PR TITLE
fix link - taken from rtd-support branch

### PR DESCRIPTION
**Pull request by dholbach**
Follow up on  #38

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -42,4 +42,4 @@ These `--git` arguments are then used to set up and configure [flux](https://www
 
 We will rely on the user installing [fluxctl](https://docs.fluxcd.io/en/latest/references/fluxctl.html#installing-fluxctl) to interact with flux directly instead of trying to replicate the functionality within `wksctl`
 
-To see a more detailed example combining Wksctl, [GitOps](https://www.weave.works/technologies/gitops/), [Ignite](https://ignite.readthedocs.io/en/stable/) also know as FireKube see [Firekube](examples/footloose/README.md#firekube-gitops)
+To see a more detailed example combining Wksctl, [GitOps](https://www.weave.works/technologies/gitops/), [Ignite](https://ignite.readthedocs.io/en/stable/) also know as FireKube see [Firekube](wks-and-footloose.md).
```
Comment (fbarl) [inline docs/get-started.md]: Don't we want to deep link to `wks-and-footloose.md#run-a-kubernetes-cluster-with-firekube` instead?

Comment (dholbach) [reply docs/get-started.md]: Sure.

Or do we want to point to https://github.com/weaveworks/wks-quickstart-firekube instead?

Comment (fbarl) [reply docs/get-started.md]: > Or do we want to point to https://github.com/weaveworks/wks-quickstart-firekube instead?

I'm not sure, the sentence starts with _To see a more detailed example..._ so I guess it should point directly to an example if possible :)

Comment (dholbach) [reply docs/get-started.md]: I didn't write the sentence.

I just wonder if we want to maintain the example doc and the full demo separately. Seems a bit much to me?

Comment (dlespiau) [reply docs/get-started.md]: I didn't write that sentence either :)

I would remove the firekube steps here and point at https://github.com/weaveworks/wks-quickstart-firekube which is more streamlined.

Comment (dholbach) [reply docs/get-started.md]: Sweet. I'll do that!

**Comment by dholbach** [comment]
Ready to review again.

**Comment by dholbach** [comment]
Fixing this in https://github.com/weaveworks/wksctl/pull/52/commits/86cdb09369c4674b3fece248c547668fdfbc095f instead. Now that the PR has progressed so far, I think we can close this one.

**Pull request closed without merging**